src/Forms/NumericField.php

```diff
@@ -232,7 +244,16 @@ public function getSchemaValidation()
      */
     public function dataValue()
     {
-        return $this->cast($this->value);
+        $value = $this->getValue();
+        if ($value === '') {
+            return null;
```

Comment:
This already happens in `cast` - we probably don't need to do it here?

Reply:
Removed